mobify/platform-scaffold
# Checkout address schema updates

This PR moves the shipping and billing address to be children of the checkout branch in the store, as well as fixing some issues with initial values for forms in the checkout

 **JIRA**: https://mobify.atlassian.net/browse/WEBDATA-69
 **Linked PRs**: n/a

## Changes
- Moved shippingAddress and billingAddress keys to the top level of the checkout state
- Fixed initial values on the shipping form
- Fixed initial values on the billing form
- Fixed missing postal code when returning to cart page after estimating shipping


## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart
- proceed to the cart
- navigate through the checkout to the payment page
- navigate back to other pages in the checkout and confirm forms are pre-filled with the correct data
- switch to the SFCC connector
- preview the [SFCC Site](https://preview.mobify.com/?url=https%3A%2F%2Fmobify-tech-prtnr-na03-dw.demandware.net%2Fon%2Fdemandware.store%2FSites-2017refresh-Site%2Fdefault%2FHome-Show&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
-  add an item to the cart
- proceed to the cart
- navigate through the checkout to the payment page
- navigate back to other pages in the checkout and confirm forms are pre-filled with the correct data
